src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -30,6 +30,7 @@
 #include "tvgLottieInterpolator.h"
 #include "tvgLottieExpressions.h"
 
+#define ROUNDNESS_EPSILON 1.0f
```

Comment:
I'm leaving a comment just in case, since the name and the number are a bit suspicious to use as the macro. 

```
//clean to understand, a obvious logic.
if (value == 0)  prepare();
if (value < 1) progress();
if (value == 1) finish();
````

```
//the numbers has been changed to text but practically, readability doesn't improved
//What is Epsilon? Readers might need to look into the EPSILON to understand the logic.
if (value == ZERO) prepare();
if (value < EPSILON) progress();
if (value == EPSILON) finish();
```
.
.
.
.
.
.
.
.
.
.

```
#define ZERO 0
#define EPSILON 1   //Ahhh. it's just 1.
```

IMO, from time to time, CONSTANT MACROS just increases the CODE longer, even it could make the maintenance worse. These guidance might help.

A. The value is **experimental number**, it can be **modifiable and it is used in several places**, allowing us to update them quickly whenever necessary. => Definitely good for maintenance.

B.  Numbers are often unexpectable magic numbers, such as PI (3.1416321321), or PATH_KAPPA (0.552284.......),  POLYSTAR_MAGIC_NUMBER(0.47829f / 0.28f) => helpful for readability.

C. People really have no idea where those complicated numbers come from, so we can apply them with explainable names =>  helpful for readability.

Reply:
so the conclusion is that you want me to remove ROUNDNESS_EPSILON?